homeassistant/components/qbittorrent/wrapper_functions.py

```diff
@@ -0,0 +1,37 @@
+"""Some recurrend functions used in the qtorrent components."""
+import logging
+
+from qbittorrent.client import Client, LoginRequired
+from requests.exceptions import RequestException
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def get_main_data_client(client: Client):
+    """Get the main data from the Qtorrent client."""
+    return client.sync_main_data()
+
+
+def create_client(url, username, password):
+    """Create the Qtorrent client."""
+    errors = {}
+    try:
+        client = Client(url)
+        client.login(username, password)
+        return client
+    except LoginRequired:
+        errors["base"] = "invalid_auth"
+        return errors
```

Comment:
Please make this function have consistent return value types. I'm not sure we need this function. The errors dict should be handled in the config flow module. We only want to catch the broad exception in the config flow, not in the integration setup.

Reply:
I was trying this approach to go around the timout exception. I will revert to my previous version.